store/store.go

```diff
@@ -191,10 +212,12 @@ func generateMongoQuery(p *Params) bson.M {
 	//if optional parameters are present, then add them to the query
 	if len(p.Types) > 0 && p.Types[0] != "" {
 		groupDataQuery["type"] = bson.M{"$in": p.Types}
+		skipParamsQuery = true
```

Comment:
If we specify the type or subType we won't filter on levels ?

Reply:
yep because I consider that filters on levels are only for blip queries. 
The reason why we do not return all the levels was to optimize blip processing. If I request a specific Type of SubType, I will not apply filtering. 
Something that we have to specify. 
Should we apply the filters on levels for all type of requests ?